### PR TITLE
feat(neuron-ui): add dynamic prmopt in wallet wizard

### PR DESCRIPTION
**Pull request by Keith-CY**
image

### CODE REVIEW

**Comment by ashchan** [comment]
That looks very nice.

File: packages/neuron-ui/src/locales/en.json [hunk 1]
```diff
@@ -47,7 +47,10 @@
       "set-wallet-name": "Give your new wallet a name",
       "set-a-strong-password-to-protect-your-wallet": "Create a strong password to protect your wallet",
       "wallet-suffix": "Wallet {{suffix}}",
-      "write-down-seed": "Write down your wallet seed and save it in a safe place"
+      "write-down-seed": "Write down your wallet seed and save it in a safe place",
+      "new-name": "Use an unused name for the new wallet",
+      "complex-password": "The password is a string of 8 to 50 characters consisting of three types of characters: uppercase letters, lowercase letters, numbers, and special symbols, and must start with a letter or a number.",
+      "same-password": "Confirm the password"
```
Comment (classicalliu) [inline packages/neuron-ui/src/locales/en.json]: This message is confusing.

Comment (Keith-CY) [reply packages/neuron-ui/src/locales/en.json]: Which one?

Comment (classicalliu) [reply packages/neuron-ui/src/locales/en.json]: value

Comment (Keith-CY) [reply packages/neuron-ui/src/locales/en.json]: How about `The password and confirm-password should be same`

Comment (classicalliu) [reply packages/neuron-ui/src/locales/en.json]: It's ok

Comment (Keith-CY) [reply packages/neuron-ui/src/locales/en.json]: updated in https://github.com/nervosnetwork/neuron/pull/799/commits/0a1c271414b23d753b54bf8d0eaf3fe08b0fd661

File: packages/neuron-ui/src/locales/en.json [hunk 2]
```diff
@@ -47,7 +47,10 @@
       "set-wallet-name": "Give your new wallet a name",
       "set-a-strong-password-to-protect-your-wallet": "Create a strong password to protect your wallet",
       "wallet-suffix": "Wallet {{suffix}}",
-      "write-down-seed": "Write down your wallet seed and save it in a safe place"
+      "write-down-seed": "Write down your wallet seed and save it in a safe place",
+      "new-name": "Use an unused name for the new wallet",
+      "complex-password": "The password is a string of 8 to 50 characters consisting of three types of characters: uppercase letters, lowercase letters, numbers, and special symbols, and must start with a letter or a number.",
+      "same-password": "The password and confirm-password should be same"
```
Comment (classicalliu) [inline packages/neuron-ui/src/locales/en.json]:
```suggestion
      "same-password": "The password and confirm password should be same"
```

**Pull request merged**